notebooks/alerts_intro_eda.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>Can you convert the commented out headers that preceded the code into markdown cells like:</p><p> </p><p><code>### on average how many alerts are seen per cluster</code></p><p> </p><p>This note applies for this cell and all below where you start with the question you are about to answer. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-alert-prediction-public/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Sure :+1: